src/LaminasRouter/ConfigProvider.php

```diff
@@ -24,8 +23,8 @@ public function getDependencies(): array
                 RouterInterface::class => LaminasRouter::class,
 
                 // Legacy Zend Framework aliases
-                \Zend\Expressive\Router\RouterInterface::class => RouterInterface::class,
-                ZendRouter::class                              => LaminasRouter::class,
+                'Zend\Expressive\Router\RouterInterface' => RouterInterface::class,
+                'ZendRouter'                             => LaminasRouter::class,
```

Comment:
This should use the previous fqcn - it became much shorter here

Reply:
🤦‍♂️